owlbot.py

```diff
@@ -98,8 +98,9 @@
     microgenerator=True,
     split_system_tests=True,
     intersphinx_dependencies={
-        "pandas": "http://pandas.pydata.org/pandas-docs/dev",
+        "dateutil": "https://dateutil.readthedocs.io/en/latest/",
         "geopandas": "https://geopandas.org/",
+        "pandas": "http://pandas.pydata.org/pandas-docs/dev",
```

Comment:
(nit, to avoid an unnecessary redirect when visiting the link)
```suggestion
        "pandas": "https://pandas.pydata.org/pandas-docs/dev",
```